Project.toml

```diff
@@ -12,8 +12,8 @@ lrslib_jll = "3873f7d0-7b7c-52c3-bdf4-8ab39b8f337a"
 
 [compat]
 Polyhedra = "0.6.14"
-julia = "1.3"
-lrslib_jll = "= 0.3.1"
+julia = "1.6"
+lrslib_jll = "0.3.1"
```

Comment:
```suggestion
lrslib_jll = "= 0.3.3"
```

Reply:
I prefer being quite strict here as a new jll version might be incompatible with the package